feat: include virtual fields in the EctoDiff struct by default. (#149)
peek-travel/ecto_diff: CHANGELOG.md

## [Unreleased][]

> - Added `:include_virtual_fields` as an option for diffing. This option allows specification of whether or not to include virtual fields in a diff. Virtual fields are not included by default. Minimum Ecto version required is 3.8.

Reviewer: ❔ minor, does this read better?
```suggestion
- Added `:include_virtual_fields` as an option for diffing. This option specifies whether to include virtual fields in a diff. Virtual fields are not included by default. Minimum Ecto version required is 3.8.
```
